apps/fluent-tester/src/TestComponents/Test.tsx

```diff
@@ -67,55 +73,52 @@ export const Test = (props: TestProps): React.ReactElement<Record<string, never>
       <Stack style={stackStyle}>
         <Text style={styles.description}>{props.description}</Text>
       </Stack>
-      {!isMobile && (
+      <ToggleButton onClick={() => setShowStatus(!showStatus)} style={[styles.statusLabel]}>
+        Platform Status {showStatus ? '-' : '+'}
```

Comment:
An FYI that our icon component supports font icons, which may be better for scaling here